public/index.html

```diff
@@ -5,10 +5,10 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
-  <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/favicons/apple-touch-icon.png">
-  <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicons/favicon-32x32.png">
-  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicons/favicon-16x16.png">
-  <link rel="mask-icon" href="%PUBLIC_URL%/favicons/safari-pinned-tab.svg" color="#e6007a">
+  <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/favicons/polkadot/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicons/polkadot/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicons/polkadot/favicon-16x16.png">
+  <link rel="mask-icon" href="%PUBLIC_URL%/favicons/polkadot/safari-pinned-tab.svg" color="#e6007a">
   <meta name="msapplication-TileColor" content="#e6007a">
   <meta name="theme-color" content="#fff">
```

Comment:
I don't think these 2 meta tags are updated when the network is switched?

Reply:
I didn't see there's any obvious differences between the background color change even they are using different color number, I will edit it